mlflow/R/mlflow/DESCRIPTION

```diff
@@ -61,7 +61,7 @@ Suggests:
     testthat (>= 2.0.0),
     xgboost
 Encoding: UTF-8
-RoxygenNote: 7.1.1
+RoxygenNote: 7.1.2
```

Comment:
Can we pip the version of `Roxygen`?

Reply:
We'd need to do the same thing again if Roxygen made another patch / minor release/